docker-compose.yml

```diff
@@ -160,10 +160,6 @@ services:
       - _APP_FUNCTIONS_SIZE_LIMIT
       - _APP_FUNCTIONS_TIMEOUT
       - _APP_FUNCTIONS_BUILD_TIMEOUT
-      - _APP_FUNCTIONS_CONTAINERS
-      - _APP_FUNCTIONS_CPUS
-      - _APP_FUNCTIONS_MEMORY
-      - _APP_FUNCTIONS_MEMORY_SWAP
```

Comment:
why did we remove these? They are being used here
https://github.com/appwrite/appwrite/pull/4650/files#diff-23222bb861952e27ab08be6b8bbb449d43186436a2a110b5a24f3bbc61c287ffR40

Reply:
Good catch!
I also updated compose.phtml with (removing swap and threshold); plus removing old docker auth variables.